framework/db/ActiveQuery.php

```diff
@@ -790,6 +790,29 @@ public function alias($alias)
         return $this;
     }
 
+
+    /**
+     * Returns conditions with alias
+     * @param array $conditions array of condition, keys to be modified
+     * @param null|string $alias set empty string for no apply alias. Set null for apply primary table alias
+     * @return array
+     */
+    public function applyTableAlias($conditions, $alias = null)
```

Comment:
But.. Its functional of ActiveQuery model, not Validator. 

Also... Some case we need the alias for build a query with alias inside child of ActiveQuery. 

May be something like this method.